plugins/inputs/nfs/nfs.go

```diff
@@ -0,0 +1,344 @@
+package nfs
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+type NFS struct {
+	Iostat   bool
+	Fullstat bool
+}
+
+var sampleConfig = `
```

Comment:
for example, it looks like iostat just gives read/write metrics, and fullstat gives many more than that. Just explain that and maybe give an idea about what sorts of metrics are included within fullstat.